Makefile

```diff
@@ -139,16 +139,20 @@ $(eval $(call MAKE_VAR_CACHE_FOR,RUNTIME_BITCODE_RELEASE_URL))
 LLVM_PATH?=$(BUILD)/llvm-static
 LLVM_CONFIG?=$(LLVM_PATH)/bin/llvm-config
 
+# Determine which flavor of the C++ standard library to link against.
+# We choose libstdc++ where available, or fallback to libc++.
+LIB_CXX_KIND?=$(shell ldconfig -p | grep -q libstdc++ && echo stdc++ || echo c++)
```

Comment:
FreeBSD doesn't have option `-p` for `ldconfig` :/
I think we have to conditionalize it based on the platform.

Reply:
Ah, that's unfortunate...

Okay, I force-pushed an amended commit that checks for `freebsd` in the platform string.

Reply:
Pushing another commit that also flips MacOS over to `libc++` in an attempt to fix the broken CI build from the last attempt.

Reply:
that works